### PR TITLE
Amount to bake

### LINKED ISSUE #399: Add note about minimum baking threshold to documentation
Per moderator on Discord there needs to be a note about the minimum staking threshold to be a baker. A user got upset that he could not bake with only 30 CCD.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To describe the transaction to see chain parameters.

## Changes

Added transaction to Transactions topic and a table describing the output. Added note to Overview of baker process with link to transaction to check stake.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

Closes #399

### CODE REVIEW

File: source/shared/net/references/transactions.rst [hunk 1]
```diff
@@ -455,3 +452,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+     - The amount of time before rewards are received
+   * - Other parameters
+     - election difficulty
+     -
+   * -
+     - foundation account index
+     -
+   * -
+     - maximum credential deployments per block
+     -
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:607]: I am guessing you would like me to fill in the details here?

Comment (abizjak) [inline source/shared/net/references/transactions.rst:607]: I am guessing you would like me to fill in the details here?

Comment (dg-concordium) [reply source/shared/net/references/transactions.rst:607]: It would be nice if we describe what they are seeing, yes.

Comment (abizjak) [reply source/shared/net/references/transactions.rst:602]: @dg-concordium I won't have time to provide details here today. I can do it by the end of the day.

Comment (abizjak) [reply source/shared/net/references/transactions.rst:602]:
```
.. list-table::
   :widths: 25 25 50
   :header-rows: 1

   * - Parameter section
     - Parameter
     - Description
   * - Parameters related to baker pools
     - Minimum equity capital
     - The minimum amount of CCD to stake to become a baker
   * -
     - maximum fraction of total stake a pool is allowed hold
     - The maximum percent of total stake any single baker pool can have
   * -
     - maximum factor a pool may stake relative to the baker's stake
     - A baker pool's stake consists of the baker's own equity capital, and
       delegated capital. This factor determines the maximum stake a baker pool
       may have relative to the equity capital. Any delegated stake above this
       threshold does not count.
   * -
     - pool owner cooldown duration
     - The amount of time the pool owner needs to wait before changes are
       effective when either decreasing stake or removing the pool. Note that
       changes are effective on the first payday after the cooldown has expired.
   * -
     - allowed range for finalization commission
     - The allowed range of finalization commissions bakers may select when
       creating or updating pools.
   * -
     - allowed range for baking commission
     - The allowed range of baking commissions bakers may select when
       creating or updating pools.
   * -
     - allowed range for transaction commission
     - The allowed range of transaction commissions bakers may select when
       creating or updating pools.
   * - Passive delegation parameters
     - finalization commission
     - The percentage of finalization rewards retained by the passive
       delegation, i.e., not given out to delegators.
   * -
     - baking commission
     - The percentage of baking rewards retained by the passive
       delegation, i.e., not given out to delegators.
   * -
     - transaction commission
     - The percentage of transaction rewards retained by the passive
       delegation, i.e., not given out to delegators.
   * - Parameters related to delegators
     - delegator cooldown duration
     - The amount of time of delegator must wait before changes are effective
       when decreasing or removing stake, or changing pools.
   * - Exchange rate parameters
     - EUR per CCD rate (approx)
     - The approximate exchange rate for the EUR to CCD exchange rate.
   * -
     - EUR per Energy rate
     - The Euro per energy rate
   * -
     - microCCD per EUR related
     - The microCCD per Euro rate.
   * - Parameters that affect rewards distribution
     - mint amount per reward period
     - The percentage increase in amount of CCD per payday.
   * -
     - mint distribution: baking reward
     - The fraction of newly minted CCD that goes towards baker rewards.
   * -
     - mint distribution: finalization reward
     - The fraction of newly minted CCD that goes towards finalization rewards.
   * -
     - transaction fee distribution: baker
     - The fraction of block transaction fees allocated to the baker.
   * -
     - transaction fee distribution: GAS account
     - The fraction of block transaction fees allocated to the GAS account.
   * -
     - GAS rewards: baking a block
     - The fraction of the GAS account that is allocated to the baker for baking
       a block.
   * -
     - GAS rewards: adding a finalization proof
     - The fraction of the GAS account that is allocated to the baker for
       including a finalization proof in a block.
   * -
     - GAS rewards: adding a credential deployment
     - The fraction of the GAS account that is allocated to the baker for
       including an account creation transaction in a block.
   * -
     - Gas rewards: adding a chain update
     - The fraction of the GAS account that is allocated to the baker for
       including an update transaction in a block.
   * - Time parameters
     - reward period length
     - The length of the reward period. All rewards are handed out at the end of
       each reward period.
   * - Other parameters
     - election difficulty
     - The election difficulty determines the probability that there is a
       block in a slot. Thus together with slot duration it determines average
       block time.
   * -
     - foundation account index
     - An index of the designated foundation reward account. The foundation
       account receives the foundation tax.
   * -
     - maximum credential deployments per block
     - The maximum amount of accounts that can be created in a block.
```

I've attempted to add short explanations. I'm not sure what the intended formatting is though.

File: source/shared/net/references/transactions.rst [hunk 2]
```diff
@@ -450,3 +447,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+
+   * - Parameter section
+     - Parameter
+     - Description
+   * - Parameters related to baker pools
+     - Minimum equity capital
+     - The minimum amount of CCD to stake to become a baker
+   * -
+     - maximum fraction of total stake a pool is allowed hold
+     - The maximum percent of total stake any single baker pool can have
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:527]:
```suggestion
     - The maximum percent of total stake any single baker pool can have.
```

File: source/shared/net/references/transactions.rst [hunk 3]
```diff
@@ -450,3 +447,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+.. list-table::
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Parameter section
+     - Parameter
+     - Description
+   * - Parameters related to baker pools
+     - Minimum equity capital
+     - The minimum amount of CCD to stake to become a baker
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:524]:
```suggestion
     - The minimum amount of CCD to stake to become a baker.
```

File: source/shared/net/references/transactions.rst [hunk 4]
```diff
@@ -450,3 +447,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+     - The percentage of baking rewards retained by the passive delegation, i.e., not given out to delegators.
+   * -
+     - transaction commission
+     - The percentage of transaction rewards retained by the passive delegation, i.e., not given out to delegators.
+   * - Parameters related to delegators
+     - delegator cooldown duration
+     - The amount of time of delegator must wait before changes are effective when decreasing or removing stake, or changing pools.
+   * - Exchange rate parameters
+     - EUR per CCD rate (approx)
+     - The approximate exchange rate for the EUR to CCD exchange rate
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:557]:
```suggestion
     - The approximate exchange rate for the EUR to CCD exchange rate.
```

File: source/shared/net/references/transactions.rst [hunk 5]
```diff
@@ -450,3 +447,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+   * -
+     - EUR per Energy rate
+     - The Euro per energy rate
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:560]:
```suggestion
     - The Euro per energy exchange rate.
```

File: source/shared/net/references/transactions.rst [hunk 6]
```diff
@@ -450,3 +447,156 @@ If you query the account information of the recipient account afterwards, it wil
...
+   * -
+     - microCCD per EUR related
+     - The microCCD per Euro rate
```
Comment (abizjak) [inline source/shared/net/references/transactions.rst:563]:
```suggestion
     - The microCCD per Euro exchange rate.
```

**Review by abizjak — APPROVED** [review]

**Pull request merged**